api-tests/features/p1-no-photo-id.feature

```diff
@@ -0,0 +1,32 @@
+@Build
+Feature: P1 No Photo Id Journey
+
+  Scenario: P1 No Photo Id Journey
+    Given I start a new 'low-confidence' journey with feature set 'p1Journeys'
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'end' event
+    Then I get a 'prove-identity-no-photo-id' page response
+    When I submit an 'next' event
+    Then I get a 'claimedIdentity' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+#      | Attribute | Values         |
+#      | context   | "hmrc_check"   |
```

Comment:
[See this comment](https://github.com/govuk-one-login/ipv-core-back/pull/2386/files#r1740991517), below, to understand why it is commented out